### PR TITLE
Sync main and develop

### PR DESCRIPTION
**Pull request by sanAkel**
This PR syncs main and develop.

@mathomp4 if all looks good, can you please approve and merge-- we probably need your _powers_! 
Thanks!

### CODE REVIEW

File: MIT_GEOS5PlugMod/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,118 @@
+esma_set_this ()
+
+set (SRCS
+  MIT_GEOS5PlugMod.F90
+  )
+
+if (NOT DEFINED MIT_CONFIG_ID)
+   message("You need to specify MIT_CONFIG_ID")
```
Comment (tclune) [inline MIT_GEOS5PlugMod/CMakeLists.txt:8]: Do you want to make this a fail message?
```suggestion
   message(FATAL_ERROR "You need to specify MIT_CONFIG_ID")
```

**Review by tclune — APPROVED** [review]
Made a CMake suggestion inline that is not crucial, but possibly worth including before hitting <merge>.

File: MIT_GEOS5PlugMod/CMakeLists.txt (hunk 1 above)
Comment (sanAkel) [reply MIT_GEOS5PlugMod/CMakeLists.txt:8]: Thanks @tclune , ✅ https://github.com/GEOS-ESM/GEOS_OceanGridComp/commit/5b21b29979951507cae695655a6000c1c3214930

**Comment by mathomp4** [comment]
@sanAkel Technically I'm not part of @GEOS-ESM/ocean-team anymore. I mean, I can push it through with superpowers, or @yvikhlya or @amolod can approve and then the button should go green

**Comment by sanAkel** [comment]
> @sanAkel Technically I'm not part of @GEOS-ESM/ocean-team anymore. I mean, I can push it through with superpowers, or @yvikhlya or @amolod can approve and then the button should go green

Of course, @mathomp4 Thanks! I'm waiting on @GEOS-ESM/ocean-team to _review_ it and I reached out to @yvikhlya.

**Review by yvikhlya — APPROVED** [review]

**Pull request merged**